filebeat/module/elasticsearch/slowlog/_meta/fields.yml

```diff
@@ -0,0 +1,61 @@
+- name: slowlog
+  description: "Slowlog events from Elasticsearch"
+  example: "[2018-06-29T10:06:14,933][INFO ][index.search.slowlog.query] [v_VJhjV] [metricbeat-6.3.0-2018.06.26][0] took[4.5ms], took_millis[4], total_hits[19435], types[], stats[], search_type[QUERY_THEN_FETCH], total_shards[1], source[{\"query\":{\"match_all\":{\"boost\":1.0}}}],"
+  type: group
+  fields:
+  - name: loglevel
+    description: "Log level"
+    example: "INFO"
+    type: keyword
+  - name: logger
+    description: "Logger name"
+    example: "index.search.slowlog.fetch"
+    type: keyword
+  - name: node_name
+    description: "Name of the node"
+    example: "v_VJhjV"
+    type: keyword
+  - name: index_name
```

Comment:
`elasticsearch.index.name`

We need to check if these two fields already exist or have to be added in Filebeat.